### ISSUE TITLE
Recipients have been deprecated and are no longer available

### ISSUE THREAD

**Issue by ayman-elmalah** [cartalyst/stripe-laravel#60]
### Laravel valet

- OS and Version: macos
- PHP Version: 6.x
- Stripe Laravel Version: 2

I need to transfer money from my account to different bank accounts
so i need to add Recipient and transfer to them
so i use this method from docs
$recipient = $stripe->recipients()->create([
    'name' => 'John Doe',
    'type' => 'individual',
]);

### Expected behaviour

* may be any success message

### Actual behaviour

* Recipients have been deprecated and are no longer available. They have been replaced by Custom accounts (see https://stripe.com/docs/connect/recipient-account-migrations), which you should use if you're creating a new platform. Please reach out at https://support.stripe.com/contact if you have any questions.

**Comment by GrahamCampbell** [comment]
I don't see how this is a bug. You just need to change your code to call the non-deprecated endpoints?

**Comment by ayman-elmalah** [comment]
@GrahamCampbell  . I'm using this package, so the function need to be changed

**Comment by GrahamCampbell** [comment]
I don't think so. The package is a direct caller to the API. You need to call a different method in this package.
